### DISCUSSION TITLE
Regarding beginner friendly issue

### DISCUSSION THREAD

**Discussion by pushker001** [vesoft-inc/nebula#4967] [General]
Hey I am new in open source can anyone guide me so I can make my first contribution

**Comment by wey-gu** [comment]
cc @Sophie-Xie we could create good first issue list :)

@pushker001, welcome to the community, what kind of contribution are you interested in, please? it's about the core of NebulaGraph(cpp) or the surrounding toolchain(java, go, spark, rust, python etc)?

**Reply by Sophie-Xie** [reply]
In these repos, you can search for the "good first issue" label and find if there are any issues you are interested in. Thanks.
